### PR TITLE
Amp/users controller tests issue 67

### PR DESCRIPTION
**Pull request by amarkpark**
Just want a second pair of eyes testing for the time being

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-    if current_user.update_attributes(user_params)
-      redirect_to edit_user_registration_path
-    end
-  end
+  # # Only users and admins can update user's profile
+  # def update
+  #   if current_user.update_attributes(user_params)
+  #     redirect_to edit_user_registration_path
+  #   end
+  # end
```
Comment (kenmazaika) [inline app/controllers/users_controller.rb]: I'd remove the code rather than comment it out.

Comment (kenmazaika) [inline app/controllers/users_controller.rb]: Looks good to me though.

**Comment by amarkpark** [comment]
The extant tests should be good for the refactored controller.  I removed some code that wasn't currently in use within the controller - this is more of a Dashboards Controller than a Users Controller and it currently grabs user data and requires authentication.  Closing issue #67

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (amarkpark) [inline app/controllers/users_controller.rb]: Yeah that was just to make reversal easier in case testing found I had broken something. ;-)

**Comment by kenmazaika** [comment]
Thumbs up, Amanda! :) :+1:

**Pull request merged**